interface_specifications/pe_openapi/apis/v1/d1_pe_definition_status_put.md

```diff
@@ -0,0 +1,37 @@
+### Status Notification to Verifier
+
+`PUT <verifier-example.io>pe/{thid}/definition/{definition_id}/status`
+
+
+#### Description
+
+This is an optional interaction and hence party can choose not to implement it. The PE REST Backend invokes an HTTP REST API exposed by the party to notify about the status of exchange.
+
+#### Parameters
+
+Path Variables:
+* `thid`: Thread ID e.g. `f1ca8245-ab2d-4d9c-8d7d-94bf310314ef`
+* `definition_id`: definition id e.g. `32f54163-7166-48f1-93d8-ff217bdb0653`
+
+Request Body
+
+```json
+{
+  "thid": "f1ca8245-ab2d-4d9c-8d7d-94bf310314ef",
```

Comment:
thid is In the path and in the body in this call

Reply:
fixed